2022-Verhagen-001/NASSA.yml

```diff
@@ -0,0 +1,38 @@
+id: 2022-Verhagen-001
+nassaVersion: 0.3.0
+moduleType: Algorithm
+title: Fertility_Rate.nlogo
```

Comment:
please make sure that the title is briefly explaining what the module does

Reply:
done